services/apps/test/apps/test_fixtures.clj

```diff
@@ -0,0 +1,45 @@
+(ns apps.test-fixtures
+  (:use [korma.db :only [create-db default-connection]])
+  (:require [clojure.string :as string]
+            [clojure.tools.logging :as log]))
+
+(def default-config-path "/etc/iplant/de/apps.properties")
+(def default-db-uri "jdbc:postgresql://dedb/de?user=de&password=notprod")
+(def default-jex-uri "https://jex-adapter:60000")
+(def default-data-info-uri "https://data-info:60000")
+(def default-notification-agent-uri "http://notificationagent:60000")
+(def default-iplant-groups-uri "http://iplant-groups:60000")
+(def default-metadata-uri "http://metadata:60000")
+(def default-permissions-uri "http://permissions:60000")
+
+(def ^:dynamic jex-uri nil)
+(def ^:dynamic data-info-uri nil)
+(def ^:dynamic notification-agent-uri nil)
+(def ^:dynamic iplant-groups-uri nil)
+(def ^:dynamic metadata-uri nil)
+(def ^:dynamic permissions-uri nil)
```

Comment:
I may end up removing these. It seemed as though it would be useful when I added them, but loading the settings from a configuration file is both necessary and easier in most cases.